.github/workflows/run_tests.yml

```diff
@@ -30,10 +30,4 @@ jobs:
           key: files-${{ hashFiles('syncode/parsers/grammars/python_grammar.lark', 'syncode/dfa_mask_store.py') }}
     - name: Run Tests
       run: |
-        python tests/test_grammar_python.py
-        python tests/test_grammar_go.py
-        python tests/test_calc.py
-        python tests/test_lr_parser.py
-        python tests/test_language_model.py
-        python tests/test_dfa_mask_python.py --independent
```

Comment:
For `test_dfa_mask_python` we were only running tokenizer independent tests. Can you do something similar? Or maybe just stop running all of these tests.